### ISSUE TITLE
Fix GitKraken installation

### ISSUE THREAD

**Issue by bddenhartog** [folkswithhats/fedy#419]
Originally reported in folkswithhats/fedy-plugins-developers#35 by @ben-lei
> Pressing the install button doesn't seem to do anything (stuck on Install). I tried to run it manually, though:
> 
> ```
> $ /usr/share/fedy/bin/run-as-root -s install.sh
>  --2016-11-11 23:06:37--  https://release.gitkraken.com/linux/gitkraken-amd64.tar.gz
> Resolving release.gitkraken.com (release.gitkraken.com)... 66.155.58.106
> Connecting to release.gitkraken.com (release.gitkraken.com)|66.155.58.106|:443... connected.
> HTTP request sent, awaiting response... 416 Requested Range Not Satisfiable
> 
>     The file is already fully retrieved; nothing to do.
> 
> gtk-update-icon-cache: Cache file created successfully.
> [Desktop Entry]
> Name=GitKraken
> Type=Application
> Icon=gitkraken
> Exec=gitkraken
> Comment=The downright luxurious Git client
> Categories=Development;IDE;
> Keywords=Git;
> StartupNotify=true
> Terminal=false
> ```
> 
> But Git Kraken isn't in my list of applications, and as mentioned Fedy doesn't change the button from "Install" to "Remove".
> 
> ```
> $ uname -a
> Linux Sunset 4.8.6-201.fc24.x86_64 #1 SMP Thu Nov 3 14:38:57 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
> ```

**Comment by bddenhartog** [comment]
Please see folkswithhats/fedy-plugins-developers#37, originally opened by @delphiki

**Comment by WolfangAukang** [comment]
I tried today to install Gitkraken through Fedy and saw that the link to the Gitkraken binary isn't being generated.

On line 18 of gitkraken.plugin, it shows this:

`ln -sf "/opt/GitKraken/gitkraken" "/usr/bin/gitkraken"`

But my opt folder shows gitkraken, all lower case. 

But before seeing this, I changed the Exec line of the desktop file, directing it to /opt/gitkraken/gitkraken, and it showed the shortcut on GNOME. Maybe this will solve.
